packages/admin-panel/src/routes/visualisations/mapOverlayGroupRelations.js

```diff
@@ -7,99 +7,95 @@ const RESOURCE_NAME = { singular: 'map overlay group relation' };
 
 export const RELATION_ENDPOINT = 'mapOverlayGroupRelations';
 
-const mapOverlayGroupCode = {
-  Header: 'Map overlay group code',
-  source: 'map_overlay_group.code',
-  editConfig: {
-    optionsEndpoint: 'mapOverlayGroups',
-    optionLabelKey: 'map_overlay_group.code',
-    optionValueKey: 'map_overlay_group.id',
-    sourceKey: 'map_overlay_group_id',
+export const FIELDS = {
```

Comment:
This is a pretty shallow refactor, just so that drilled-down views which also need to edit map overlay group relations can use the same source of truth for these fields

**If** we want to keep this, I would certainly do the same thing for **dashboard items**, whose fields (I imagine) are also used when drilled down from **dashboards**

Probably not worth immediately refactoring _all_ resource pages to have a `FIELDS` object, but it’s busywork I’m more than happy to do if you like?

Reply:
I did do this PR **without** this refactor, so if you don’t think it makes sense to have this different pattern just for two pages, I just need to revert one commit